<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="sideBar" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #2C3E50;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnDashboard">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: white;
    border: none;
    text-align: left;
    padding-left: 15px;
}
QPushButton:hover {
    background-color: #34495E;
}
QPushButton:pressed {
    background-color: #2980B9;
}</string>
         </property>
         <property name="text">
          <string>Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCallLogs">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: white;
    border: none;
    text-align: left;
    padding-left: 15px;
}
QPushButton:hover {
    background-color: #34495E;
}
QPushButton:pressed {
    background-color: #2980B9;
}</string>
         </property>
         <property name="text">
          <string>Call Logs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLiveCalls">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: white;
    border: none;
    text-align: left;
    padding-left: 15px;
}
QPushButton:hover {
    background-color: #34495E;
}
QPushButton:pressed {
    background-color: #2980B9;
}</string>
         </property>
         <property name="text">
          <string>Live Calls</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTimeTime">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: white;
    border: none;
    text-align: left;
    padding-left: 15px;
}
QPushButton:hover {
    background-color: #34495E;
}
QPushButton:pressed {
    background-color: #2980B9;
}</string>
         </property>
         <property name="text">
          <string>Time to Time</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="pageDashboard">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Dashboard Page</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageCallLogs">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="labelTotalCalls">
          <property name="text">
           <string>Total Connected Calls: 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string>Call Ref ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Channel</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Call Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Call Time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Duration</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageLiveCalls">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="labelLiveCallsStatus">
          <property name="text">
           <string>Active Calls: 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableLiveCalls">
          <column>
           <property name="text">
            <string>Channel</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Channel Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Call Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Call Time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Caller ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Called ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Listen</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
